### PR TITLE
Fix data synchronization logs

### LINKED ISSUE #566: Data syncronization logs aren't being displayed
The global `logrus` logger is being unwittingly overridden with a custom schema logger, preventing any future plain text logs from being displayed.

In layman terms: the `rclone` progress logs can't be seen by users. 😱

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
Closes #566 and redirects `rclone` notices to the `[DEBUG]` level.

### CODE REVIEW

**Comment by 0x2b3bfa0** [comment]
### Bonus point: add a photo of an angry walrus

**Review by DavidGOrtega — APPROVED** [review]
👍
